packages/components/src/interfaces/internal/CollectionCard.ts

```diff
@@ -2,14 +2,15 @@ import type { ImageProps } from "~/interfaces"
 import type { IsomerSiteProps, LinkComponentType } from "~/types"
 
 export interface Tag {
-  title: string
+  label: string
```

Comment:
this is as discussed a categorical tag right? shouldn't it be having additional properties:

```
export interface Tag {
    category: string
    values: string[]
}
```

Reply:
oops, saw the changes in the BE PR